__tests__/app.test.js

```diff
@@ -322,3 +322,30 @@ describe('DELETE /api/comments/:comment_id', () => {
       })
   })
 })
+
+describe('GET /api/users', () => {
+  test('200: gets all users in an array', () => {
+    return request(app)
+      .get('/api/users')
+      .expect(200)
+      .then((response) => {
+        const users = response.body.users;
+        expect(users.length).toBeGreaterThan(0);
+        users.forEach((user) => {
+          expect(user).toEqual(expect.objectContaining({
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String)
+          }));
+        });
+      });
+  });
+  test('404: not found with bad path', () => {
+    return request(app)
+    .get('/api/invalid')
```

Comment:
You don't need to do this test here. As long as you've done it previously it's fine. It should just be in it's own describe block for all not found errors. It's not specific to any endpoint. 👍